src/lib/image/mount/mount.c

```diff
@@ -56,7 +57,7 @@ int singularity_image_mount(int argc, char ** argv) {
     }
 
     singularity_priv_init();
-    singularity_config_open(joinpath(SYSCONFDIR, "/singularity/singularity.conf"));
+    singularity_config_init(joinpath(SYSCONFDIR, "/singularity/singularity.conf"));
```

Comment:
We can move remove all the config_init calls from `lib/image/*/*.c` and simply move it into one call in `src/simage.c`